app/controllers/new_gestionnaire/procedures_controller.rb

```diff
@@ -76,5 +77,11 @@ def ensure_ownership!
         redirect_to root_path
       end
     end
+
+    def redirect_to_avis_if_needed
```

Comment:
Pourquoi on a besoin de ça ?

Reply:
Effectivement, c'est un point à débattre.

Ca permet au gens qui se loggue pour rendre un avis alors qu'il n'ont pas été affecté à une procédure, d'arriver directement sur la page avis.

Je pense que cette logique devrait se retrouver à terme dans la partie routage du log in.

Reply:
Yes, probablement mieux dans le login qu'ici.

On peut d'ailleurs se poser la question de distinguer ou non les pages / controlleurs de login / signup selon les roles

Reply:
ok, vu en direct, on met un commentaire pour dire de bouger la logique apres la bascule totale vers le nouveau design